plugins/Toolbox/src/CloudSync/SubscribedPackagesModel.py

```diff
@@ -59,15 +62,20 @@ def initialize(self, subscribed_packages_payload: List[Dict[str, Any]]) -> None:
                 "md5_hash": item["md5_hash"],
                 "is_dismissed": False,
             }
-            if self._sdk_version not in item["sdk_versions"]:
-                package.update({"is_compatible": False})
-            else:
-                package.update({"is_compatible": True})
+
+            package.update({"is_compatible": self._is_any_version_compatible(package_manager, item["sdk_versions"])})
+
             try:
                 package.update({"icon_url": item["icon_url"]})
             except KeyError:  # There is no 'icon_url" in the response payload for this package
                 package.update({"icon_url": ""})
             self._items.append(package)
         self.setItems(self._items)
 
-
+    @staticmethod
+    def _is_any_version_compatible(package_manager: PackageManager, api_versions: [str]) -> bool:
+        """:return: True when any of the provided api versions is compatible"""
+        for version in api_versions:
+            if package_manager.isPackageCompatible(Version(version)):
+                return True
+        return False
```

Comment:
I'll accept this now but just something to consider; this can be simplified to:

    any(package_manager.isPackageCompatible(Version(version)) for version in api_versions)

Could be considered more semantic (since it literally uses a function called `any`). Could also be considered harder to read since it's a longer line.